plugins/in_calyptia_fleet/in_calyptia_fleet.c

```diff
@@ -2099,19 +2112,8 @@ static int load_fleet_config(struct flb_in_calyptia_fleet_config *ctx)
         if (exists_cur_fleet_config(ctx) == FLB_TRUE) {
             return execute_reload(ctx, cur_fleet_config_filename(ctx));
         }
-        else if (exists_new_fleet_config(ctx) == FLB_TRUE) {
```

Comment:
The new file may not be valid so we should never use it when starting up. Behavior should be, in order:
1. Use cur if that exists.
2. Use old if that exists.
3. Wait to pull a valid config from the backend.